ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -625,7 +625,13 @@ def ocs_install_verification(
             object_store_user = defaults.EXTERNAL_CLUSTER_OBJECT_STORE_USER
             realm = config.ENV_DATA.get("rgw-realm")
             host, user, password, ssh_key = get_external_cluster_client()
-            external_cluster = ExternalCluster(host, user, password, ssh_key)
+            external_cluster = ExternalCluster(
+                host,
+                user,
+                password,
+                ssh_key,
+                jump_host=config.EXTERNAL_MODE.get("ssh_jump_host"),
+            )
```

Comment:
if we are considering the changes according to IBM cloud, do we need changes according to IBM config too ?

Reply:
This change requires following additional configuration for the environments where it will be required (probably in the external cluster config or some secret file). We don't want to add this to the platform related config in ocs-ci.

```
EXTERNAL_MODE:
  ssh_jump_host:
    host: "..."
    user: "..."
    private_key: "..."
    password: "..."
```

Reply:
Shouldn't i expect similar changes in ocs_ci/deployment/ibm.py ??

Reply:
And there is any usage of `ExternalCluster` class there?